### PR TITLE
Add PLEP 1 on purpose and guidelines of PLEPs

### PR DESCRIPTION
**Pull request by namurphy**
The purpose of this pull request is to establish the procedure and guidelines for creating and revising PlasmaPy Enhancement Proposals (PLEPs).

Parts of this document are adapted from:
 - [APE Purpose and Process](https://github.com/astropy/astropy-APEs/blob/master/APE1.rst) (for Astropy Proposals for Enhancement)
 - [SEP Purpose and Guidelines](https://github.com/sunpy/sunpy-SEP/blob/master/SEP-0001.md) (for SunPy Enhancement Proposals)
 - [PEP Purpose and Guidelines](https://www.python.org/dev/peps/pep-0001/) (for Python Enhancement Proposals)

### CODE REVIEW

File: PLEP-0001.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+Minor changes to a PLEP that do not affect its meaning or intent
+(e.g., formatting changes, fixes to typos, updates to links, minor
+rewording to improve clarity, and reversing cosmic ray bit flips) do
+not need to go through the full review process, but do require
+approval by the Coordinating Committee.
+
+### Submitting a PLEP
+
+All new PLEPs and amendments to PLEPs should be submitted as pull
+requests into the PlasmaPy-PLEP repository.  The pull request may be
```
Comment (SolarDrew) [inline PLEP-0001.md]: may -> should? Depending on how strongly we want to encourage the transparency.

File: PLEP-0001.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+PLEP's editor.  The editor is responsible for aiding the sponsor by
+making sure that the PLEP follows the accepted standard and
+facilitating communication between the sponsor and the Coordinating
+Committee.  The editor and sponsor may be the same person.
+
+When the editor agrees that the PLEP is reasonably close to the
+accepted standard, then the sponsor, the editor, or another member of
+the Coordinating Committee should email the PlasmaPy list to introduce
+the PLEP and request community feedback.  The discussion should take
+place through normal communication channels.  Important points and
```
Comment (SolarDrew) [inline PLEP-0001.md]: I appreciate these may change, but is it worth saying here what the 'normal communication channels' are? Or perhaps linking to some kind of 'normal contact channels' page on the website, which we would keep up to date.

File: PLEP-0001.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+discussion should ideally continue until a general consensus among the
+PlasmaPy community has been reached.  
+
+The Coordinating Committee is tasked with making the final decision on
+the PLEP.  A PLEP shall be accepted when approved by at least
+two-thirds of the Coordinating Committee.
+
+### PLEP status
+
+The status of a standard or process PLEP may be any of the following:
```
Comment (SolarDrew) [inline PLEP-0001.md]: Can an informational PLEP have status or is it omitted?

File: PLEP-0001.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+  incompatible change to the application programming interface (API).
+  A standard PLEP will start out as a proposal and eventually evolve
+  into a design document if accepted.
+
+* A **process** PLEP describes a new process or change to an existing
+  process in the management and coordination of PlasmaPy.  Examples
+  include changes to PlasmaPy decision-making processes or management
+  structure, guidelines, or procedures.  A process PLEP will start out
+  as a proposal and eventually evolve into a document on the
+  governance of PlasmaPy.
```
[the remark this answers is not in the document]
Comment (namurphy) [reply PLEP-0001.md]: There is a [PLEP template](https://github.com/PlasmaPy/PlasmaPy-PLEPs/blob/master/PLEP-template.md) but that's more for standard PLEPs.  Do you think we should have separate standard PLEP and process PLEP templates?

File: PLEP-0001.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+A PLEP may come about when a contributor has an idea for how to
+improve PlasmaPy or when a Coordinating Committee member or subpackage
+maintainer requests a PLEP to be written before a pull request is
+accepted. The following subsections describe the procedure for PLEPs
+to be created, submitted, decided upon, amended, and implemented.
+Anyone who abides by the code of conduct may submit a PLEP.
+
+### Creating a PLEP
+
+Before writing a PLEP, it is generally advisable
```
[the remark this answers is not in the document]
Comment (namurphy) [reply PLEP-0001.md]: Yep, my usual way of writing is to start sentences and then forget to finish them. 😖 I was probably going to say it's advisable to bring this up in discussions on Matrix, etc., I think?

File: PLEP-0001.md (hunk 3 above)
Comment (namurphy) [reply PLEP-0001.md]: I was wondering about this.  The status for informational enhancement proposals for other projects is usually "Discussion" but this doesn't seem quite the best way to go about it.  Maybe omitting the status of informational PLEPs would be best.

File: PLEP-0001.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (SolarDrew) [reply PLEP-0001.md]: Not necessarily - I'm just not clear on how a PLEP is different once it becomes a 'document on the governance of PlasmaPy'. I guess what I really mean to ask is is it the same document or once it's accepted are we then going to reconstitute it into a more formal document somewhere else?

File: PLEP-0001.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (SolarDrew) [reply PLEP-0001.md]: Seems reasonable :) Bring it up on Matrix, check it's not a duplicate, get informal feedback, all that good stuff.

File: PLEP-0001.md (hunk 3 above)
Comment (SolarDrew) [reply PLEP-0001.md]: Maybe. But then explicit is better than implicit, right? Perhaps just 'information' would do?

**Comment by StanczakDominik** [comment]
To keep things localised, from yesterday's telecon:

- [ ] Double check that licenses for PLEP repository  include SunPy and Astropy licenses [Nick]
    - [ ] https://github.com/PlasmaPy/PlasmaPy-PLEPs/blob/master/LICENSE.md
- [ ] Allow PLEP voting by CC members approving pull requests on GitHub
    - [ ] Note about voting by email or in meetings as described in PLEP 2ish

**Comment by namurphy** [comment]
The Astropy-APEs repository does not yet have a license, so I created #19 instead as a placeholder.

@StanczakDominik @SolarDrew @tulasinandan - Shall we go ahead and finally officially approve this?  I vote in support!

**Comment by StanczakDominik** [comment]
Let the voting commence!

:+1:

**Comment by namurphy** [comment]
:+1:

**Comment by namurphy** [comment]
@SolarDrew @StanczakDominik @tulasinandan - A friendly reminder to vote on PLEPs 1, 2, 4, 5, & 7 when you have a chance!

**Comment by tulasinandan** [comment]
TNP approves these changes!

**Comment by namurphy** [comment]
Thanks everyone!  We have a PLEP!

**Pull request merged**